src/main/java/com/myMoneyTracker/dao/AppUserDao.java

```diff
@@ -14,4 +19,7 @@
 @Repository
 public interface AppUserDao extends JpaRepository<AppUser, Long>{
 
+	@Query("SELECT u FROM AppUser u WHERE u.username = ?1")
```

Comment:
This query is needed because the format of the method (if you want JpaRepository to take the column from the method name) should be "findBy{Column}", not "findUserBy{Column}"